### PR TITLE
Examples of the lib usage added

### PR DESCRIPTION
**Pull request by nkaskov**
* Fixed a bug with the private mod ffi
* Polynomial evaluation example added
* Simple real numbers example added
* Installation instruction updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,64 @@ The library is under development and the ETA of the first version is set to Q2 2
...
+
+On Debian systems, everything can be installed with the following command:
+
+```bash
+sudo apt install build-essential libssl-dev cmake clang-12 git
+```
+
+## Installation process
+
+1. Build and install OpenFHE library. Right now you need to use the Fair Math fork. It contains required features, which will be included in the next planned  release (v1.1.5):
```
Comment (HovsepPapoyan) [inline README.md:35]: space before release

File: examples/simple_real_numbers.rs [hunk 2]
```diff
@@ -0,0 +1,96 @@
+extern crate openfhe;
+use cxx::{CxxVector, SharedPtr};
+use openfhe::ffi as ffi;
+
+fn main(){
```
Comment (HovsepPapoyan) [inline examples/simple_real_numbers.rs:5]: Braces starting from the new line for consistency

File: README.md (hunk 1 above)
Comment (nkaskov) [reply README.md]: Fixed, thanks!

File: examples/polynomial_evaluation.rs [hunk 3]
```diff
@@ -0,0 +1,112 @@
+extern crate openfhe;
+use cxx::{CxxVector, SharedPtr};
+use openfhe::ffi as ffi;
+
+fn main()
+{
+    use std::time::Instant;
```
[the remark this answers is not in the document]
Comment (nkaskov) [reply examples/polynomial_evaluation.rs:7]: Fixed as well

File: examples/simple_real_numbers.rs (hunk 2 above)
Comment (nkaskov) [reply examples/simple_real_numbers.rs]: Fixed

**Review by HovsepPapoyan — APPROVED** [review]

**Pull request merged**